config/manager/manager.yaml

```diff
@@ -76,10 +76,14 @@ spec:
         env:
         - name: PEERPODS_NAMESPACE
           value: "openshift-sandboxed-containers-operator"
-        - name: KATA_MONITOR_IMAGE
+        - name: RELATED_IMAGE_KATA_MONITOR
           value: quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-monitor:latest
         - name: SANDBOXED_CONTAINERS_EXTENSION
           value: kata-containers
+        - name: RELATED_IMAGE_CAA
+          value: registry.redhat.io/openshift-sandboxed-containers/osc-rhel9-cloud-api-adaptor
+        - name: RELATED_IMAGE_CAA_WEBHOOK
+          value: registry.redhat.io/openshift-sandboxed-containers/osc-rhel9-cloud-api-adaptor-webhook
```

Comment:
It is a bit unclear how these two are consumed. I cannot find users for this in the repo.

Also, are the default values appropriate for upstream ?